cpu/saml21/include/periph_cpu.h

```diff
@@ -29,12 +29,45 @@ extern "C" {
 /**
  * @brief   Mapping of pins to EXTI lines, -1 means not EXTI possible
  */
+#if defined(CPU_MODEL_SAML21E18A) || defined(CPU_MODEL_SAML21E18B) || \
+    defined(CPU_MODEL_SAML21E17A) || defined(CPU_MODEL_SAML21E17B) || \
+    defined(CPU_MODEL_SAML21E16A) || defined(CPU_MODEL_SAML21E16B) || \
+    defined(CPU_MODEL_SAML21E15A) || defined(CPU_MODEL_SAML21E15B)
+static const int8_t exti_config[1][32] = {
+    { 0,  1,  2,  3,  4,  5,  6,  7, -1,  9, 10, 11, -1, -1, 14, 15,
+      0,  1,  2,  3, -1, -1,  6,  7, 12, 13, -1, 15, -1, -1, 10, 11},
+};
+#elif defined(CPU_MODEL_SAMR30E18A)
+static const int8_t exti_config[1][32] = {
+    { 0,  1, -1, -1,  4,  5,  6,  7, -1,  9, -1, -1, 12, 13, 14, 15,
+      0,  1,  2,  3, -1, -1, -1, -1, 12, 13, -1, 15,  8, -1, 10, 11},
+};
```

Comment:
PA00, PA01, PA04, PA05, PA12, PA13 do not exist on SAMR30E
PA10, PA11, PA20 exist on SAMR30E (internally connected to RF)